tests/e2e-tests/specs/admin/create-shipping-label.test.js

```diff
@@ -18,150 +18,72 @@ describe( 'Create shipping label', () => {
   	  	  	dialog.accept();
 		} );
-		await loginAndConfirmExtensionActivation();
-		console.log( 'I am logged in!' );
-		await createSimpleProduct();
-		console.log( 'Simple product is created!' );
-
-		// Go to general settings page
-		await StoreOwnerFlow.openSettings( 'general' );
-
-		// Set base location with state CA.
-		await expect( page ).toSelect( 'select[name="woocommerce_default_country"]', 'United States (US) — California' );
-		// Sell to all countries
-		await expect( page ).toSelect( '#woocommerce_allowed_countries', 'Sell to all countries' );
-		// Set currency to USD
-		await expect( page ).toSelect( '#woocommerce_currency', 'United States (US) dollar ($)' );
-		// Tax calculation should have been enabled by another test - no-op
-		// Save
-		await settingsPageSaveChanges();
-		console.log( 'General settings have been set and saved!' );
-
-		// Verify that settings have been saved
-		await Promise.all( [
-			expect( page ).toMatchElement( '#message', { text: 'Your settings have been saved.' } ),
-			expect( page ).toMatchElement( 'select[name="woocommerce_default_country"]', { text: 'United States (US) — California' } ),
-			expect( page ).toMatchElement( '#woocommerce_allowed_countries', { text: 'Sell to all countries' } ),
-			expect( page ).toMatchElement( '#woocommerce_currency', { text: 'United States (US) dollar ($)' } ),
-		] );
-
-		// Enable COD payment method
-		await StoreOwnerFlow.openSettings( 'checkout', 'cod' );
-		await setCheckbox( '#woocommerce_cod_enabled' );
-		await settingsPageSaveChanges();
-
-		// Verify that settings have been saved
-		await verifyCheckboxIsSet( '#woocommerce_cod_enabled' );
-		console.log( 'Payment settings have been set and saved!' );
-
-		// Add missing WooCommerce pages
-		await StoreOwnerFlow.openStatus( 'tools' );
-		await clickReactButton( '.button.install_pages' );
-		await page.waitForSelector( '.updated.inline', { text: 'All missing WooCommerce pages successfully installed' } );
-		console.log( 'Missing WooCommerce pages installed!' );
-
-
-		await StoreOwnerFlow.logout();
-		console.log( 'I am logged out!' );
 	} );
 
-	it( 'should add item to cart and display cart item in order review', async () => {
-		await CustomerFlow.goToShop();
-		console.log( 'I am on the shop page!' );
-		await CustomerFlow.addToCartFromShopPage( 'Simple product' );
-
-		console.log( 'Product added to the cart!' );
-		await CustomerFlow.goToCheckout();
-		console.log( 'I am on the checkout page!' );
-		await CustomerFlow.productIsInCheckout( 'Simple product', 1, 9.99 );
-		console.log( 'Product is in the order review page!' );
-		await expect( page ).toMatchElement( '.cart-subtotal .amount', { text: '$9.99' } );
-	} );
+	it( 'should create a new shipping label', async () => {
+		await db.loadFixtures( async ( models ) => {
+			await StoreOwnerFlow.login();
+			await StoreOwnerFlow.openExistingOrderPage( models.wp_posts[1].ID );
 
-	it( 'should allow customer to place order', async () => {
```

Comment:
I would leave this as a not connected test step for future use. We might end up having product page customizations when WCS is enabled.